### PR TITLE
Add tests for rhpam70-authoring-ha template

### PR DESCRIPTION
**Pull request by sutaakar**

### CODE REVIEW

File: framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/DeploymentScenarioBuilderFactory.java [hunk 1]
```diff
@@ -35,6 +36,7 @@
     WorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder getWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder();
     KieServerWithExternalDatabaseScenarioBuilder getKieServerWithExternalDatabaseScenarioBuilder();
     KieServerWithDatabaseScenarioBuilder getKieServerWithDatabaseScenarioBuilder();
+    ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilder getClusteredWorkbenchKieServerDatabasePersistentScenarioBuilder();
```
Comment (jhrcek) [inline framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/DeploymentScenarioBuilderFactory.java]: Just to clarify: what does the work `Persistent` in scenario mean? Doesn't presence of `Database` already imply persistence?

Comment (sutaakar) [reply framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/DeploymentScenarioBuilderFactory.java]: @jhrcek Not necessarily. Persistent means that the deployment state is persisted through pod restarts. Theoretically database doesn't imply persistence as there may be ephemeral databases (though not used in our templates).

File: framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/ClusteredWorkbenchKieServerDatabasePersistentScenarioImpl.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        super.deploy();
+
+        logger.info("Processing template and creating resources from " + OpenShiftTemplate.CLUSTERED_WORKBENCH_KIE_SERVER_DATABASE_PERSISTENT.getTemplateUrl().toString());
+        envVariables.put(OpenShiftTemplateConstants.IMAGE_STREAM_NAMESPACE, project.getName());
+        project.processTemplateAndCreateResources(OpenShiftTemplate.CLUSTERED_WORKBENCH_KIE_SERVER_DATABASE_PERSISTENT.getTemplateUrl(), envVariables);
+
+        workbenchDeployment = new WorkbenchDeploymentImpl(project);
+        workbenchDeployment.setUsername(DeploymentConstants.getWorkbenchUser());
+        workbenchDeployment.setPassword(DeploymentConstants.getWorkbenchPassword());
+        workbenchDeployment.scale(1);
```
Comment (jakubschwan) [inline framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/ClusteredWorkbenchKieServerDatabasePersistentScenarioImpl.java]: If workbench is clustered, shouldn't we want default count of replicas > 1 ?

Comment (sutaakar) [reply framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/ClusteredWorkbenchKieServerDatabasePersistentScenarioImpl.java]: I set it to one to keep the same approach in all scenarios. That way all the tests can expect the same configuration available.

Comment (jakubschwan) [reply framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/ClusteredWorkbenchKieServerDatabasePersistentScenarioImpl.java]: Ok, make sense

**Comment by sutaakar** [comment]
Test failure is related to image version

**Pull request merged**